src/main/java/org/openstreetmap/atlas/geography/atlas/dynamic/policy/DynamicAtlasPolicy.java

```diff
@@ -59,9 +95,16 @@ public DynamicAtlasPolicy(final Function<Shard, Optional<Atlas>> atlasFetcher,
         };
     }
 
-    public Shard getInitialShard()
+    public Set<Shard> getInitialShards()
+    {
+        return this.initialShards;
+    }
+
+    public MultiPolygon getInitialShardsBounds()
```

Comment:
Nit: getInitialShardsBounds --> getInitialShardBounds. (no s)

Reply:
I added the "s" there because there are multiple shards. Let me know if that reads too weird and I'll remove.

Reply:
Sounds better without it. Your call thought :)